lib/features/auth/view/widget/title_element.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+
+class TitleElement extends StatelessWidget {
+  const TitleElement({
+    super.key,
+    required this.name,
+    this.paddingBottom = 0,
+    required this.color,
+    required this.fontSize,
+    this.fontWeight,
+    this.width,
+  });
+
+  final String name;
+  final double paddingBottom;
+  final Color color;
+  final double fontSize;
+  final FontWeight? fontWeight;
+  final double? width;
+
+  @override
+  Widget build(BuildContext context) {
+    final child = Padding(
+      padding: EdgeInsets.only(bottom: paddingBottom),
+      child: Center(
+        child: Text(
+          name,
+          style: TextStyle(
+            color: color,
+            fontSize: fontSize,
+            fontWeight: fontWeight ?? FontWeight.normal,
+          ),
+          textAlign: TextAlign.center,
+        ),
+      ),
+    );
+    return width != null ? SizedBox(width: width, child: child) : child;
```

Comment:
I think it is ok to set the width of a sized box to null; it will act as if you didn't pass it.
If this gives a specific effect, then it is ok.